contracts/deployments/base/.migrations.json

```diff
@@ -4,6 +4,8 @@
   "003_base_vault_and_token": 1722429523,
   "004_super_oeth": 1723219797,
   "005_mutlisig_harvester": 1723220782,
+  "006_base_amo_strategy": 1724882452,
   "007_bridged_woeth_strategy": 1724168481,
-  "008_oethb_zapper": 1724651792
+  "008_oethb_zapper": 1724651792,
+  "006_base_amo_strategy": 1724882095
```

Comment:
nit: Duplicate entry in the file. I believe you added one manually, can you remove it?

Reply:
I pushed a fix for this here: https://github.com/OriginProtocol/origin-dollar/pull/2206/commits/0bfdebca12e34b07f9a432d25cce700e143e4baf

Reply:
tnx